drivers/clk/clk.c

```diff
@@ -344,6 +344,7 @@ unsigned int __clk_get_enable_count(struct clk *clk)
 {
 	return !clk ? 0 : clk->core->enable_count;
 }
+EXPORT_SYMBOL_GPL(__clk_get_enable_count);
```

Comment:
Clock enable count should not be known by clients, if you need this it sometimes implies you are using the API incorrectly.